config/zones/GL.yaml

```diff
@@ -1 +1,3 @@
 timezone: America/Nuuk
+region: Americas
+country: GL
```

Comment:
As a fellow dane I am very disappointed in you on this one 😉 🇬🇱

Reply:
I used the ISO-3166-1 (https://github.com/lukes/ISO-3166-Countries-with-Regional-Codes/blob/master/all/all.csv) standard to generate the configs, but it hurts my danish heart 😭